aws-databricks/variables.tf

```diff
@@ -0,0 +1,51 @@
+
+variable "vpc_id" {
+  description = "VPC ID"
+  default     = null
+}
+
+variable "aws_region" {
+  description = "AWS Region"
+  default     = null
+}
+
+variable "databricks_account_username" {
+  description = "Databricks Account Username"
+}
+variable "databricks_account_password" {
+  description = "Databricks Account Password"
+}
+variable "databricks_account_id" {
+  description = "Databricks Account ID"
+}
+
+variable "tags" {
+  default = {}
+}
+
+variable "workspace_name" {
+  description = "Name of the workspace"
+  default     = "test"
```

Comment:
Internal/core modules should avoid defaults unless it's to prevent breaking changes or something. They should rely solely on external values and have no internal opinionation IMO